salt/metalk8s/archives/unmounted.sls

```diff
@@ -0,0 +1,31 @@
+{# Remove mounted ISOs which were removed from configuration #}
```

Comment:
I thought about it, but right now we can't, because we want to first mount new ISOs, reconfigure the repo container, then unmount old ISOs (otherwise we may have issues with it, I think).

Reply:
Right make sense